RayLin.md

```diff
@@ -1,13 +1,17 @@
-# 11/01 ~ 11/02
+# 11/08 ~ 11/10
 
 - [Video Control]
   - Bug 1271765 - Visual refresh of media controls
-    - landing
+    - target on next release cycle
+    - latest try: https://treeherder.mozilla.org/#/jobs?repo=try&revision=c890fe464ff2a4c13505db9d1e137fe752d90f5e&selectedJob=30963545
   - Bug 1302320 - add a test to confirm that video control show "noaudio" icon when video has no audio
     - patch done and pushed. Will add review flag once Bug 1271765 landed
   - Bug 1311700 - Add test to confirm that video control show controls in different sizes correctly
+    - patch done and pushed. Will add review flag once Bug 1271765 landed
+  - Bug 1312334 - add test to confirm that every control work fine when open media directly
     - WIP
-    - get intermittent error while selecting element and inspecting its size
+  - Add a test to confirm error state and error text should display and fit on most video size
+    - discuss with Alastor, our first diagnosis is that we can not produce error state manually. The `mError` is an internal used variable, and not exposed to any API. Since we could not easily to simulate state like network error or decode aborted in test environment, remove this test from plan.
```

Comment:
Possibly for decoding and unsupported error. 
For network error, the hard part is to simulate internet disconnection prior to video is fully buffered. Also other error states rely on different conditions like the present of MIME header or if download was aborted by browser, etc. Perhaps, instead of comprehensively test on each error states, we could just create a minimal test case to confirm the correctness on specific problem for now, e.g. `Add a test to confirm that error message is show while opening unsupported format media`